assets/tests/unit/home.spec.js

```diff
@@ -1,87 +1,204 @@
-import { shallowMount } from '@vue/test-utils'
-import BucketList from '@/components/BucketList.vue'
-jest.mock('axios')
+import { shallowMount } from '@vue/test-utils';
+import AchievedGoals from '@/components/AchievedGoals.vue';
+import BucketList from '@/components/BucketList.vue';
+jest.mock('axios');
+import axios from 'axios';
 
 beforeEach(() => {
-  jest.resetModules()
-  jest.clearAllMocks()
+  jest.resetModules();
```

Comment:
Could be useful to add a linter.

Reply:
Fixed in 4d7753036e69ff5c64180b21fa7ba4e2957357d8